Index: includes/context-admin.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/includes/context-admin.inc,v
retrieving revision 1.14
diff -u -p -r1.14 context-admin.inc
--- includes/context-admin.inc	11 Oct 2010 22:18:22 -0000	1.14
+++ includes/context-admin.inc	19 Jan 2011 23:20:53 -0000
@@ -102,7 +102,9 @@ function ctools_context_add_argument_for
   $args = ctools_get_arguments();
   $choices = array();
   foreach ($args as $name => $arg) {
-    $choices[$name] = $arg['title'];
+    if (empty($arg['no ui'])) {
+      $choices[$name] = $arg['title'];
+    }
   }
 
   asort($choices);
Index: page_manager/plugins/tasks/page.admin.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/page_manager/plugins/tasks/page.admin.inc,v
retrieving revision 1.30
diff -u -p -r1.30 page.admin.inc
--- page_manager/plugins/tasks/page.admin.inc	5 Jan 2011 22:35:46 -0000	1.30
+++ page_manager/plugins/tasks/page.admin.inc	19 Jan 2011 23:20:53 -0000
@@ -1114,7 +1114,9 @@ function page_manager_page_argument_form
 
   $options = array();
   foreach ($plugins as $id => $plugin) {
-    $options[$id] = $plugin['title'];
+    if (empty($plugin['no ui'])) {
+      $options[$id] = $plugin['title'];
+    }
   }
 
   asort($options);
Index: plugins/arguments/nid.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/plugins/arguments/nid.inc,v
retrieving revision 1.10
diff -u -p -r1.10 nid.inc
--- plugins/arguments/nid.inc	29 Jan 2010 20:18:01 -0000	1.10
+++ plugins/arguments/nid.inc	19 Jan 2011 23:20:53 -0000
@@ -20,6 +20,7 @@ $plugin = array(
     '#type' => 'textfield',
     '#description' => t('Enter the node ID of a node for this argument'),
   ),
+  'no ui' => TRUE,
 );
 
 /**
Index: plugins/arguments/term.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/plugins/arguments/term.inc,v
retrieving revision 1.12
diff -u -p -r1.12 term.inc
--- plugins/arguments/term.inc	31 Dec 2010 21:31:50 -0000	1.12
+++ plugins/arguments/term.inc	19 Jan 2011 23:20:53 -0000
@@ -21,6 +21,7 @@ $plugin = array(
   'settings form' => 'ctools_term_settings_form',
   'placeholder form' => 'ctools_term_ctools_argument_placeholder',
   'breadcrumb' => 'ctools_term_breadcrumb',
+  'no ui' => TRUE,
 );
 
 /**
Index: plugins/arguments/uid.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/plugins/arguments/uid.inc,v
retrieving revision 1.12
diff -u -p -r1.12 uid.inc
--- plugins/arguments/uid.inc	29 Jan 2010 20:18:01 -0000	1.12
+++ plugins/arguments/uid.inc	19 Jan 2011 23:20:53 -0000
@@ -24,6 +24,7 @@ $plugin = array(
   'default' => array('to_arg' => TRUE),
   'path placeholder' => '%pm_uid_arg', // This is in pagemanager.
   'path placeholder to_arg' => TRUE,
+  'no ui' => TRUE,
 );
 
 /**
Index: plugins/arguments/vid.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/ctools/plugins/arguments/vid.inc,v
retrieving revision 1.6
diff -u -p -r1.6 vid.inc
--- plugins/arguments/vid.inc	29 Jan 2010 20:18:01 -0000	1.6
+++ plugins/arguments/vid.inc	19 Jan 2011 23:20:53 -0000
@@ -21,6 +21,7 @@ $plugin = array(
     '#type' => 'textfield',
     '#description' => t('Enter the vocabulary ID for this argument'),
   ),
+  'no ui' => TRUE,
 );
 
 /**
